presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -6049,8 +6049,8 @@ public void testDropStatsPartitionedTable()
                         "('p_bigint', null, 0.0, 0.0, null, null, null), " +
                         "(null, null, null, null, 0.0, null, null)");
 
-        // Drop stats for the entire table (partition_values = NULL)
-        assertUpdate(format("CALL system.drop_stats('%s', '%s', NULL)", TPCH_SCHEMA, tableName));
+        // Drop stats for the entire table
+        assertUpdate(format("CALL system.drop_stats('%s', '%s')", TPCH_SCHEMA, tableName));
```

Comment:
consider leaving test with NULL to assert change is backward compatible.

Reply:
I could, but i don't think it is worth it, especially that it was not documented :)

Reply:
It is here: https://github.com/prestosql/presto/pull/3940/files#diff-dc25bafb3bcbf06b1786a0453ba14515R969-R970.

Reply:
indeed, thanks.